browser/ui/brave_pages.cc

```diff
@@ -92,4 +92,8 @@ void ShowWalletSettings(Browser* browser) {
   ShowSingletonTabOverwritingNTP(browser, GURL(kWalletSettingsURL));
 }
 
+void ShowAppsPage(Browser* browser) {
+  ShowSingletonTabOverwritingNTP(browser, GURL(chrome::kChromeUIAppsURL));
```

Comment:
This works well from private(tor) window? Maybe it should be opened in normal window even it's requested from private window's app menu.